model/schema.json

```diff
@@ -82,6 +82,10 @@
           "description": "The action id.",
           "type": "string"
         },
+        "input_type": {
+          "description": "The input type of the original action.",
+          "type": "string"
+        },
```

Comment:
The `input_type` name is unclear to me, without reading the description I thought the `action_result` could have an actual input_type. But this is actually an `ActionInputType`, Is there hard requirement to keep that name?

Reply:
no hard requirements, it's just a copy of the `input_type` field from the .fleet-actions, thus it has the same name.
If you have better suggestions for the name let me know, can adjust to whatever name feels better.

Reply:
`ActionInputType` looks ok to me, I just want to make sure we don't forget it.
Is there any test we could add to protect that feature from any change?

Reply:
sounds good, will update to the ```action_input_type```, will check on the test coverage

Reply:
Thanks @aleksmaus